__tests__/PaginationBoxView-test.js

```diff
@@ -1474,4 +1474,198 @@ describe('Test custom props', () => {
       ).toBe('Item 1');
     });
   });
+  describe('prevPageRel/nextPageRel/selectedPageRel', () => {
+    it('should render default rel if not defined', function () {
```

Comment:
Last request: add a test to ensure we can disable the `rel` all together by setting them to `undefined`